_component_design/progress-indicators.md

```diff
@@ -3,7 +3,7 @@ title: Progress Indicators
 parent: Processing
 layout: component
 category: Components
-usage: The Helix 12-column grid provides structure for website content.
+usage: The Helix 12 Column Grid provides structure for website content.
 preview-image: components-grid.svg
 need: progress-indicators
 status: unknown
```

Comment:
Also, if the resource is listed as false, the page is not visible to the world. You can skip these as they're still in a state of flux or may be removed completely.